.github/workflows/remark.yml

```diff
@@ -36,6 +36,13 @@ jobs:
     - name: Check *.md files
       run: git ls-files -z '*.md' | xargs -0 -n 1 -I {} ./node_modules/.bin/remark {} -u lint -f > /dev/null
 
+    - name: Linkcheck book
+      run: |
+        rustup default nightly
+        rustup toolchain install nightly --component rust-docs
```

Comment:
GHA has a stable Rust toolchain installed by default. Does it also work with that or does it require nightly for some reason?

Reply:
According to linkcheck.sh: "This requires the rust-docs rustup component to be installed in the nightly
# toolchain."